### DISCUSSION TITLE
When is the distribution for Windows planned?

### DISCUSSION THREAD

**Discussion by SergeyKGoncharov** [Chia-Network/chia-blockchain#7284] [Support]
When is the distribution for Windows planned?

**Comment by loppefaaret** [comment]
we have windows versions for download on our release page ?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
